pilot/pkg/xds/ads.go

```diff
@@ -511,6 +518,13 @@ func (s *DiscoveryServer) initConnection(node *core.Node, con *Connection) error
 	if s.StatusGen != nil {
 		s.StatusGen.OnConnect(con)
 	}
+	if s.XEventing != nil {
+		// TODO: more info - actual instance of istiod, timestamp, etc.
+		s.XEventing.PodEvent(con.proxy.ConfigNamespace, con.proxy.Metadata.InstanceName,
+			"istio-connect",
+			fmt.Sprintf("id=%s", con.ConID), false)
```

Comment:
nit: in other messages we have `time:%s`, here we have `id=%s`. Probably we want consistent format

Reply:
Right, will fix it.